### ISSUE TITLE
How to load model with a custom metric

### ISSUE THREAD

**Issue by cicorias** [cicorias/njit-deeplearn-explain-shap#1]
load model with custom metrics:

```

    # HACK TODO Remove  results\models\model20201115-123712.h5
    # HACK https://github.com/tensorflow/tensorflow/issues/33646#issuecomment-566433261 
    from tensorflow.keras.models import load_model
    custom_objects={'F1Score':F1Score()}
    model_temp = load_model(cfg['PATHS']['MODEL_WEIGHTS'] + 'model20201115-123712.h5', custom_objects=custom_objects, compile=False)

```
